### PR TITLE
Update colors

### PR DESCRIPTION
**Pull request by jcassarly**
Heres a quick list of what I changed
EFEFEF [body, nav] -> E3E8EF - Tower City
045184 [nav text] -> 53595A - Steelyard Gray
3588CC [nav hover] -> 1371AC - edgewater blue
095790 [links in index.html] -> 023251 - Uptown Blue
095790 [faq] -> 1371AC - edgewater blue
C0C0C0 [not active] -> AFB9BC - Cleveland sky
1D86C8 [about, .reg, footer] -> 0F5D8D - Blue Line

### CODE REVIEW

File: www/index.html [hunk 1]
```diff
@@ -45,7 +46,7 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
 				<h3>HackCWRU brings together the brightest and most creative students to promote unrestricted technological innovation.</h3>
 				<h3>Hosted at Case Western Reserve University, over 350 students will spend 24 hours bringing their imagination into the real world.  Attendees will work with peers and mentors to create projects in one of six project tracks. Stay tuned for more on our track system!</h3>
 				<h3>Don't be intimidated, however.  You don't have to know what you're doing to attend.  If you are new to hackathons, we look forward to introducing you to a world of creation.</h3>
-				<h3>Lastly, at HackCWRU we abide by and enforce <a style="text-decoration: none; color:#095790;" href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH's Code of Conduct</a>.</h3>
+				<h3>Lastly, at HackCWRU we abide by and enforce <a style="text-decoration: none; color:#023251;" href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH's Code of Conduct</a>.</h3>
```
Comment (ajm188) [inline www/index.html]: can we move this style to a CSS class?

File: www/index.html [hunk 2]
```diff
@@ -135,7 +136,7 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
 				</div>
            </div>-->
 			<div style="text-align:center; font-size:22px;">
-				<p><a style="color:#095790;" target="_blank" href="sponsor.pdf">Learn more about sponsoring HackCWRU</a><br>Interested?  Email <a style="color:#095790;" href="mailto:yxh597@case.edu">Yidi</a>!</p>
+				<p><a style="color:#023251;" target="_blank" href="sponsor.pdf">Learn more about sponsoring HackCWRU</a><br>Interested?  Email <a style="color:#023251;" href="mailto:yxh597@case.edu">Yidi</a>!</p>
```
Comment (ajm188) [inline www/index.html]: ditto about classes here

File: www/index.html [hunk 3]
```diff
@@ -6,7 +6,8 @@
 		<meta name="description" content="HackCWRU 2017 hosted at Case Western Reserve University. Sign up for email updates!">
 		<meta name="author" content="HackCWRU">
 		<link href="site.css" rel="stylesheet">
-		<link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:200' rel='stylesheet' type='text/css'>
+		
+		<link href='http://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
```
Comment (ajm188) [inline www/index.html]: what was the ":200" for and why did it get removed from the query param?

**Review by ajm188 — CHANGES_REQUESTED** [review]

File: www/index.html (hunk 3 above)
Comment (jcassarly) [reply www/index.html]: I dont actually know what that was, I didnt notice that when I was doing the editting, so ill just  add the ":200" after "Open+Sans"

**Comment by jcassarly** [comment]
I think i made the changes requested.  I am not sure if the updates were actually applied to the pull request.

Also, I dont have a lot of experience with css and html, so it would be great if someone could make sure I created the css classes and used them right

File: www/index.html [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (ajm188) [inline www/index.html]: you've got some leftovers of a git conflict here

File: www/index.html [hunk 5]
```diff
@@ -153,3 +154,157 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
 		</footer>
 	</body>
 </html>
+=======
```
Comment (ajm188) [inline www/index.html]: and here

File: www/index.html [hunk 6]
```diff
@@ -153,3 +154,157 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
...
+		<link href="site.css" rel="stylesheet">
+		<link href='http://fonts.googleapis.com/css?family=Open+Sans:200' rel='stylesheet' type='text/css'>
+	</head>
+
+	<body>
+		<a id="mlh-trust-badge" style="position:fixed;top:0;right:50px;max-width:100px;width:10%;min-width: 60px;display:block;z-index:10000" href="https://mlh.io/seasons/s2016/events?utm_source=s2016&utm_medium=TrustBadge&utm_campaign=s2016" target="_blank"><img src="https://s3.amazonaws.com/logged-assets/trust-badge/s2016.png" alt="MLH Official - Spring 2016" style="width:100%;" ></a>
+			<ul class="nav">
+				<li><a href="#home"><img class="img-hover" src="Logo_Hat.svg" style="height:40px;margin-bottom:-10px;"/></a></li>
+				<li><a href="#about">About</a></li>
+                <!--				<li><a href="#schedule">Schedule</a></li>-->
```
Comment (ajm188) [inline www/index.html]: can we remove this comment?

File: www/index.html [hunk 7]
```diff
@@ -153,3 +154,157 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
...
+		<div id="about">
+			<div id="home_about_text">
+				<h5>About HackCWRU</h5>
+				<h3>HackCWRU brings together the brightest and most creative students to promote unrestricted technological innovation.</h3>
+				<h3>Hosted at Case Western Reserve University, over 150 students will spend 36 hours bringing their imagination into the real world.  Attendees will work with peers, mentors, company sponsors, and a whole suite of rapid prototyping equipment.  From 3D printers to PCB routers to fully blown wood and metal shops, Sears think[box] has it all.</h3>
+				<h3>Don't be intimidated, however.  You don't have to know what you're doing to attend.  If you are new to hackathons, we look forward to introducing you to a world of creation.</h3>
+				<h3>Lastly, at HackCWRU we abide by and enforce <a class="link_text_1" href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH's Code of Conduct</a>.</h3>
+			</div>
+		</div>
+        <!--<div id="schedule">
```
Comment (ajm188) [inline www/index.html]: ditto, remove this commented out junk

File: www/index.html [hunk 8]
```diff
@@ -153,3 +154,157 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
...
+				<img draggable="false" src="scroll-pictures/6.jpg"/>
+				<img draggable="false" src="scroll-pictures/2.jpg"/>
+				<img draggable="false" src="scroll-pictures/3.jpg"/>
+			</div>
+		</div>
+		<div id="sponsors">
+			<div style="text-align:center;">
+				<h5>Sponsors and Partners</h5>
+			</div>
+           <!--<div class="home_sponsors_logos">
```
Comment (ajm188) [inline www/index.html:118]: remove comments

File: www/site.css [hunk 9]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (ajm188) [inline www/site.css]: git conflict leftovers (GCL)

File: www/index.html [hunk 10]
```diff
@@ -153,3 +154,157 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
...
+				<ul class="smedia">
+					<li><a target="_blank" href="https://www.facebook.com/HackCWRU/"><img class="smlogo img-hover" src="sm-logos/facebook.png"/></a></li>
+					<li><a target="_blank" href="https://twitter.com/hackcwru"><img class="smlogo img-hover" src="sm-logos/twitter.png"/></a></li>
+					<li><a href="mailto:hackcwru@case.edu"><img class="smlogo img-hover" src="sm-logos/mail.png"/></a></li>
+				</ul>
+			</div>
+		</footer>
+	</body>
+</html>
+>>>>>>> 29aae13... added ":200" back to link and moved some styling to css classes
```
Comment (ajm188) [inline www/index.html]: git conflict leftovers

File: www/site.css [hunk 11]
```diff
@@ -366,3 +367,384 @@ div.box {
  }
 
 }
+=======
```
Comment (ajm188) [inline www/site.css]: GCL

File: www/site.css [hunk 12]
```diff
@@ -366,3 +367,384 @@ div.box {
...
+@-o-keyframes "bannermove" {
+ 0% {
+   margin-left: 0px;
+ }
+ 100% {
+   margin-left: -5566px;
+ }
+
+}
+>>>>>>> 29aae13... added ":200" back to link and moved some styling to css classes
```
Comment (ajm188) [inline www/site.css]: GCL

**Review by ajm188 — CHANGES_REQUESTED** [review]
I think you messed up your git somehow, I would talk to someone at GB today to help you out, or find me in the slack

**Comment by jcassarly** [comment]
I fixed the git conflicts, not sure what happened there

The comments were there before Is tarted updating colors, I think they were commented out because we were not able to use those parts of the website yet, i.e. we haven't put up the schedule

**Comment by jcassarly** [comment]
There are actually still some issues with the colors that i didnt realize, so I will go fix that

**Comment by jcassarly** [comment]
The colors should be fixed and I added the logo on Frontify currently

File: www/Logo.svg [hunk 13]
```diff
@@ -1,63 +1 @@
...
...09.07-16.2c0-4.06,0-8.12,0-11.17,0-1.77,0-3,0-3.89l6.9,1.16v9.89h12.33v-7.82l11.52,1.94v60.67h-76V334.72l12.34-2v7.84H286v-9.85l7.56-1.24v19.77h-9.64v13.24h22.65Zm-3.41-33.56c0.08,2.95,0,18.66,0,30.1H287.33v-6.33H297V328.88l3.07-.5Zm-1.36-165.1,19.92,10.42-11.9,75.11-8-3.48v-82ZM337.6,265.16L301.77,276V249.61Zm17.54,33.92-49.83-20.51,36.31-11Zm-119.2,92.76a10.3,10.3,0,0,1,0-20v20ZM201.39,413.9a7.43,7.43,0,1,1,7.43-7.43,7.43,7.43,0,0,1-7.43,7.43m-3.5,1h0ZM213,457.06l1.83,2.34H188l2.12-2.34H213Zm172.36-3.46-7.23,9.25h41.67l-8.38-9.25h-7.56V416a10.87,10.87,0,0,0-10-19.31L375,385a13.7,13.7,0,0,0-12.46-16.93V338.92L374,340.85v-34L359.7,301l-15.79-36.82-30.86-13.39,12.42-78.44L300.05,159,273.5,172.28l12.31,78.87-30.54,13-16.58,36.82-14.83,5.9v34l12.08-2v29.41A13.7,13.7,0,0,0,225,385l-18.83,11.68a10.87,10.87,0,0,0-10,19.31V453.6H188.6l-8.38,9.25h41.67l-7.23-9.25H206V416.31a10.82,10.82,0,0,0,6-12.2L230.36,393a13.68,13.68,0,0,0,5.58,2.36v55.89h13v11.59H350.24V451.26h12.32V395.55a13.67,13.67,0,0,0,7.07-2.54L388,404.12a10.83,10.83,0,0,0,6,12.2V453.6h-8.65Z" transform="translate(-75.09 -71.29)"/></g><polygon class="cls-3" points="327.01 343.65 327.01 343.65 327.01 343.65 327.01 343.65"/></svg>
```
Comment (ajm188) [inline www/Logo.svg]: missing newline at EOF

File: www/index.html [hunk 14]
```diff
@@ -17,7 +17,7 @@
 		<ul class="nav">
 			<li><a href="#home"><img class="img-hover" src="Logo_Hat.svg" style="height:40px;margin-bottom:-10px;"/></a></li>
 			<li><a href="#about">About</a></li>
-              <!--				<li><a href="#schedule">Schedule</a></li>-->
```
Comment (ajm188) [inline www/index.html]: either delete this comment, or revert this change; git churn--

File: www/index.html [hunk 15]
```diff
@@ -45,7 +45,7 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
 				<h3>HackCWRU brings together the brightest and most creative students to promote unrestricted technological innovation.</h3>
 				<h3>Hosted at Case Western Reserve University, over 350 students will spend 24 hours bringing their imagination into the real world.  Attendees will work with peers and mentors to create projects in one of six project tracks. Stay tuned for more on our track system!</h3>
 				<h3>Don't be intimidated, however.  You don't have to know what you're doing to attend.  If you are new to hackathons, we look forward to introducing you to a world of creation.</h3>
-				<h3>Lastly, at HackCWRU we abide by and enforce <a style="text-decoration: none; color:#095790;" href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH's Code of Conduct</a>.</h3>
+				<h3>Lastly, at HackCWRU we abide by and enforce <a class="link_text_1" href="http://static.mlh.io/docs/mlh-code-of-conduct.pdf">MLH's Code of Conduct</a>.</h3>
```
Comment (ajm188) [inline www/index.html]: can we use more descriptive class names?

File: www/index.html [hunk 16]
```diff
@@ -136,7 +136,7 @@ <h3 id="cwru">Case Western Reserve University<br></h3>
 				</div>
            </div>-->
 			<div style="text-align:center; font-size:22px;">
-				<p>Learn more about sponsoring HackCWRU<br>Interested?  Email <a style="color:#095790;" href="mailto:bgr19@case.edu">Ben</a>!</p>
+				<p>Learn more about sponsoring HackCWRU<br>Interested?  Email <a class="link_text_2" href="mailto:bgr19@case.edu">Ben</a>!</p>
```
Comment (ajm188) [inline www/index.html]: ditto on descriptive class names

File: www/site.css [hunk 17]
```diff
@@ -1,368 +1,377 @@
-body {
```
Comment (ajm188) [inline www/site.css:1]: please revert the indentation-only changes to this file

**Review by ajm188 — CHANGES_REQUESTED** [review]

File: www/site.css (hunk 17 above)
Comment (jcassarly) [reply www/site.css:1]: I am not sure exactly what to do here, should I change the tabs to spaces?

**Comment by jcassarly** [comment]
I changed the image back as per Yidi's request

File: www/site.css (hunk 17 above)
Comment (ajm188) [reply www/site.css:1]: Yes, we use spaces for indentation

Comment (jcassarly) [reply www/site.css:1]: ok, not sure why that was, I think that the file I cloned was like that

Comment (ajm188) [reply www/site.css:1]: no, most likely a side effect of your editor rewriting the file

**Comment by ajm188** [comment]
@jcassarly if we're not merging this, is it safe for me to delete the branch?

**Comment by jcassarly** [comment]
Should be

On Apr 10, 2017 2:40 PM, "Andrew Mason" <notifications@github.com> wrote:

@jcassarly <https://github.com/jcassarly> if we're not merging this, is it
safe for me to delete the branch?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/hacsoc/HackCWRU2016-Website/pull/34#issuecomment-293038578>,
or mute the thread

**Comment by ajm188** [comment]
Deleted

**Pull request closed without merging**